google/codelab/create_codelab_image_wrapper.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
Does this really need to have so much logic in it?
I thought the helper library was structured to make this easier. This looks like it is all boilerplate except you are uploading a couple custom files and executing a custom script against it.

Maybe because this is bash it isnt structured properly to let you inject that in. If you see a simple refactoring in the helper libraries that would let . you do that, then most of this logic can be eliminated which will be safer and easier to maintain.

Reply:
I'll take a look to see if there's any way I can reduce the logic here.  For context, all this logic was directly in the Jenkins job prior to my change---I'm just checking in what was on the Jenkins job (with a few small changes) so that it's managed by source control.  I'll take a look to see if there's any reasonable way to clean this up.

Reply:
I was able to clean up this logic somewhat by making use of some of the existing functions in the helper library (and pushing some others into that library).